flipper-plugin-realm/package.json

```diff
@@ -79,6 +79,7 @@
     "prettier": "^2.7.1",
     "react": "latest",
     "react-dom": "latest",
+    "realm": "^11.3.1",
```

Comment:
Realm is added to get the types, correct?

Reply:
yes!

Reply:
Should be just fine as a devDependency